source/content/guides/decoupled/drupal-backend-starters/06-configure-existing.md

```diff
@@ -0,0 +1,176 @@
+subtitle: Configure an Existing Backend Project
+description: Learn how to configure an existing backend project to use a frontend starter kit.
+tags: [webops, workflow, decoupled]
+contributors: [backlineint, cobypear, hckia, whitneymeredith]
+layout: guide
+showtoc: true
+permalink: docs/guides/decoupled/drupal-backend-starters/configure-existing
+anchorid: configure-existing
+contenttype: [guide]
+innav: [true]
+categories: [create]
+cms: [decoupled]
+audience: [development]
+product: [decoupled]
+integration: [--]
+---
+
+Pantheon offers a [backend starter project](/guides/decoupled/drupal-backend-starters/create) to simplify the process of configuring a Drupal site for use with our Front-End Site starter kits. However, you can configure an existing backend project to use one of Pantheon's Front-End Site starter kits.
+
+## Before You Begin
+
+These instructions assume that you have already installed Drupal using your
+preferred method.
+
+Configuration requirements vary depending on the features you intend to use within your frontend starter kit. The instructions below are broken down into three related sections.
+
+## Configure Basic Builds
+
+Follow the steps below if you only need to source content anonymously from your Drupal site.
+
+### Add and Enable Dependencies
+
+1. Run the following Composer commands:
+
+    ```bash{promptUser: user}
+    composer config minimum-stability alpha
+    composer require drupal/jsonapi_menu_items drupal/jsonapi_hypermedia drupal/decoupled_router
+    ```
+
+1. Enable the `media_library`, `jsonapi`, `jsonapi_menu_items_hypermedia`, and `decoupled_router` modules. This also enables a number of dependencies.
+
+1. Clear the Drupal cache.
+
+### Configure Images
+
+The Front-End Site starter kits assume that you are using Drupal's core Media module to manage images. You must make one of two possible adjustments if you are using the default image field instead:
+
+- Add a new field of type `Media` to the Article content type. The resulting
+  field should have the label `Media Image`. The machine name will be `field_media_image`. Be sure to rebuild Drupal's cache after adding this field.
+- Update your starter kit to use the default image field instead of the Media
+  field.
+
+### Ensure That Your Content Uses URL Aliases
+
+The starter kit uses the Decoupled Router module to determine the path where your Drupal content should display. As a result, while the pattern used to define your paths does not matter, your page and article content must have a path alias of some kind. This can be defined manually during content creation, or automatically using the [Pathauto module](https://www.drupal.org/project/pathauto).
+
+### Create Supporting Content
+
+The Front-End Sites starter kits assume that there is at least one published article and page in your Drupal backend. If your site does not have any article or page content, you should create some before proceeding.
```

Comment:
Is this a roundabout way of saying "the starterkits don't fail gracefully if there isn't content" ?